src/client/pages/DataExport/CountrySelect/CountrySelect.tsx

```diff
@@ -21,7 +24,15 @@ const CountrySelect: React.FC = () => {
   const countries = useDataExportCountries()
```

Comment:
Since that hook is used elsewhere, I'll check those changes won't create unexpected behavior first. But I think that's a better approach to encapsulate that logic.

Reply:
I was checking whether it would affect something else or not, and seems it won't. I'll add the filter logic an the hook and test now

Reply:
Added it. Thanks for the suggestions 👌